components/Common/Banner/index.module.scss

```diff
@@ -0,0 +1,28 @@
+.banner {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
```

Comment:
Use tailwind variables for padding and gap please.
Use tailwind variables for flexboxes and justifying content please 👀 

Tailwind can do everything you're doing here, so I rather that you use it's variables because it's less manual CSS.

Reply:
I have use "basic" css because my understanding of thing it's we just use tailwind for variable. But I think I misunderstood 🤔

Reply:
Tailwind is an utility-first CSS framework. Meaning that we don't use it just for colours but any specific values that can be reusable.

This reduces greatly of having different things with random different values and creates a way of streamlining the aspects of a component layout.

Reply:
thanks for clarification.